tests/integration/sliders/test_sliders.py

```diff
@@ -159,7 +159,7 @@ def test_slsl005_slider_tooltip(dash_dcc):
                         style=dict(height=100),
                     ),
                 ],
-                style=dict(maxHeight=300, overflowX="scroll"),
+                style=dict(maxHeight=300, overflowX="scroll", width=400),
```

Comment:
Slider tooltips don't render correctly on Percy. They do pretty well when the width is locked like this, though there's still a bit of an offset. But when the width is set by the page they're way off: Most of the slider, and its handles, are positioned in percentages but the tooltips are positioned in pixels. Ironic, because the tooltips and handles could have used the same positioning logic: `left: <some %>; transform: translateX(-50%)`. But when Percy picks this up it rerenders the page with a different width and pixels and % no longer have the same relationship at all.

I suspect there's a real bug here, perhaps if your slider is subject to CSS scaling or if you change its width through a style prop after rendering; but I haven't tested it. Sliders do update correctly when their width updates as the window is resized though, which is the most important use case.